src/services/mentees.js

```diff
@@ -58,6 +59,20 @@ module.exports = class MenteesHelper {
 				}
 			}
 
+			let communications = null
+			if (cachedProfile?.meta?.communications_user_id) {
+				try {
+					const chat = await communicationHelper.login(id, tenantCode)
+					communications = chat
+				} catch (error) {
+					console.error('Failed to log in to communication service:', error)
+				}
+			}
+
+			cachedProfile.meta = {
+				...cachedProfile.meta,
+				communications,
+			}
```

Comment:
Unnecessary reassignment, with a risk of accidentally overwriting `meta.communications` when login fails.